python/dpu_utils/utils/chunkwriter.py

```diff
@@ -39,7 +41,13 @@ def __init__(self, out_folder: Union[RichPath, str], file_prefix: str, max_chunk
         self.__max_chunk_size = max_chunk_size
         self.__file_suffix = file_suffix
 
-        self.__num_files_written = 0  # type: int
+        self.__mode = mode.lower()
+        assert self.__mode == 'a' or self.__mode == 'w', 'Mode must be either append (a) or write (w). Given: {0}'.format(mode)

```

Comment:
```suggestion
        assert self.__mode in { 'a', 'w' }, 'Mode must be either append (a) or write (w). Given: {0}'.format(mode)
```
(maybe slightly more clear)